packages/build/src/core/normalize_flags.ts

```diff
@@ -102,6 +102,8 @@ const getDefaultFlags = function ({ env: envOpt = {} }, combinedEnv) {
     tracing: {
       enabled: false,
       apiKey: '-',
+      // defaults to always sample
```

Comment:
does `sampleRate = 1` not mean `no sample`?

Reply:
No, it means always sample, 0 means `don't sample` and anything else will lead to `1/sampleRate` ratio of sampling:

- https://github.com/honeycombio/honeycomb-opentelemetry-node/blob/a2da97851f6d443a8ac4fafe3844c38cb8912b7c/src/deterministic-sampler.ts#L24-L33